legadilo/static/css/base.css

```diff
@@ -2,6 +2,8 @@
   --content-footer-height: 104px;
   --content-mobile-footer-height: 248px;
   --content-header-height: 56px;
+  --bs-body-bg: #f8f9fa; /* Light mode */
```

Comment:
I don’t think we should redefine colors. Removing the `bg-*` classes on `footer` and updating the behavior of `.content-footer` should be enough:
```css
.content-footer {
  height: var(--content-mobile-footer-height);
  background-color: var(--bs-dark);
}

[data-bs-theme="dark"] .content-footer {
  background-color: var(--bs-tertiary-bg);
}
```

This way, we have the same footer as today on light (which I like) and on dark the footer has the same color than the navbar. What do you think?

Note: with your current modification, footer on light theme is broken.